src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--month-length--29">
+      <div class="calendar__day calendar__day--first__day-sat"></div>
```

Comment:
You should not hardcode the first day and the number of days in the month. Instead, use SCSS @for to generate the required number of divs.